src/sigma/params.cpp

```diff
@@ -10,6 +11,13 @@ Params* Params::get_default() {
         //fixing generator G;
         GroupElement g("9216064434961179932092223867844635691966339998754536116709681652691785432045",
                        "33986433546870000256104618635743654523665060392313886665479090285075695067131");
+
+        if((::Params().NetworkIDString() == CBaseChainParams::MAIN)) {
```

Comment:
`::Params().GenConsensus().IsMain()` is the preferred way across the code now

Reply:
Ok thank you will change.